### PR TITLE
Flags for Sweep

### PR DESCRIPTION
**Pull request by bonham79**
Pull request from test.

### CODE REVIEW

File: baseline/sweep [hunk 1]
```diff
@@ -14,25 +14,37 @@ readonly PATIENCE=12
 readonly SED_EM_ITERATIONS=10
 readonly MAX_ENSEMBLE_SIZE=10
 
+LANGS=""
```
Comment (kylebgorman) [inline baseline/sweep:17]: One last question: shouldn't this be defined lower? E.g., right before the loop on line 112 or so?

File: baseline/sweep [hunk 2]
```diff
@@ -42,41 +54,96 @@ train() {
...
+                        rm -rf ${OUTPUT}
+                        break
+                else
+                    echo "This will delete all prior data. If you wish to continue, press y."
+                fi
+            done
+            ;;
+    esac
+done
+if [[ ! -d "${OUTPUT}" ]]
```
Comment (kylebgorman) [inline baseline/sweep]: You can just do `mkdir -p` which is the same logic as this if/then.

File: baseline/sweep [hunk 3]
```diff
@@ -42,41 +54,96 @@ train() {
...
+                echo "${OUTPUT}/"*"/${lang}"
+                find ${OUTPUT}/* -type d -name ${lang} -exec rm -rf {} +
+            done
+            ;;
+        r)  
+            # Resets entire run. This will delete all prior data. 
+            echo "This will delete all prior data. If you wish to continue, press y."
+            while : ; do
+                read -n 1 k <&1
+                if [[ $k = y ]] ;
```
Comment (kylebgorman) [inline baseline/sweep]: Put the `then` on the same line as the conditional, like we do with `do`.

File: baseline/sweep [hunk 4]
```diff
@@ -42,41 +54,96 @@ train() {
...
+            done
+            ;;
+        r)  
+            # Resets entire run. This will delete all prior data. 
+            echo "This will delete all prior data. If you wish to continue, press y."
+            while : ; do
+                read -n 1 k <&1
+                if [[ $k = y ]] ; 
+                    then
+                        printf "\nResetting run."
```
Comment (kylebgorman) [inline baseline/sweep]: Suggest:

```
echo
echo Resetting run.

File: baseline/sweep [hunk 5]
```diff
@@ -42,41 +54,96 @@ train() {
...
+done
+if [[ ! -d "${OUTPUT}" ]]
+    # For starting a new run.
+    then
+        mkdir ${OUTPUT}
+fi
+
 main
+
+
```
Comment (kylebgorman) [inline baseline/sweep]: Please remove extra blanks here.

File: baseline/sweep (hunk 1 above)
Comment (bonham79) [reply baseline/sweep:17]: it's referenced in the functions so throws an unbound variable error

Comment (kylebgorman) [reply baseline/sweep:17]: Ahh okay makes sense.

**Review by kylebgorman — APPROVED** [review]

**Comment by kylebgorman** [comment]
@bonham79 I had to do a manual merge---could you quickly eyeball that to make sure it's kosher before I merge the PR itself?

**Comment by bonham79** [comment]
looks good from what i can see

**Pull request merged**